fuzzbench/conftest.py

```diff
@@ -11,25 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Fake jobs."""
+"""A pytest conftest.py file that defines fixtures."""
+import redis
 
```

Comment:
no need for blank line.
And pytest should come before redis. Both are 3rd party modules.

Reply:
Done.